WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -243,9 +243,8 @@ function(sechdr) {
         htmlArr[idx++] = "<div id='_upComingTaskListHdr'>";
         htmlArr[idx++] = "<table width=100% class='DwtListView-Column'><tr>";
         this.dId = Dwt.getNextId();
-        htmlArr[idx++] = "<td><div class='DwtListHeaderItem-label ";
-        htmlArr[idx++] = ZmTaskListView.SEC_COLOR[sechdr];
```

Comment:
But looks like we are removing the support for color completely. I would suggest to clear the object as well if its not used . If in future its required, it can always be taken from history.